### PR TITLE
Log err.message instead of <unavailable> err object

### PR DESCRIPTION
**Pull request by Eselce**
Most of the log messages printing a caught error object `err` (or `chrome.runtime.lastError`) were converted to  `<unavailable>`, which is not very helpful. But then `err.message` provides the information needed.

### CODE REVIEW

**Comment by Sxderp** [comment]
As an aside, I think it depends on which console you're looking at. There're three consoles; extension debugging, browser, and page. One of them is bound to actually provide access to the object.

**Comment by Eselce** [comment]
@Sxderp Yes, that's correct. I played around with exceptions, errors, etc., and there are quite a few occations, when you get a full output on the (tool) `Ctrl-Shift-I` console (provided, that the filter is set approriately), even with a stack trace. Many useful logs go to the browser (web) `Ctrl-Shift-J` console, where access is limited. Debugging can provide the object, but that has nothing to do with these log messages.

Maybe there is a case, where we would like to see the whole object (and it has a nice output), but for me, it looks as if I often miss a helpful error message, if an error occurs. Tell me, if there are really many cases (apart from debugging with break points and value evaluation), where you need the whole error object (in the case of `err = chrome.runtime.lastError`, documentation tells me, that only `err.message` is guaranteed)...

**Comment by arantius** [comment]
Please help me understand/test:  Clear reproduction case where the effect before this change is undesirable?

File: src/bg/user-script-registry.js [hunk 1]
```diff
@@ -105,8 +105,8 @@ async function installFromDownloader(userScriptDetails, downloaderDetails) {
     }
     return details.uuid;
   }).catch(err => {
-    console.error('Error in installFromDownloader()', err);
-    // Rethrow so caller can also deal with it
+    console.error('Error in installFromDownloader():', err.message);
```
[the remark this answers is not in the document]
Comment (Eselce) [reply src/bg/user-script-registry.js]: This could be a good idea for every such error message (except for ```chrome.runtime.lastError```).

**Comment by arantius** [comment]
Sorry I forgot about this for a while, now it won't merge cleanly.

**Comment by Cerberus-tm** [comment]
@arantius Thank you for all these recent fixes. Very nice.

**Pull request merged**